### ISSUE TITLE
Support for jupyter notebooks

### ISSUE THREAD

**Issue by frankhart2018** [bndr/pipreqs#199]
I was using pipreqs for a project and I found out that if libraries are imported inside a jupyter notebook, then they aren't being taken into account by pipreqs while creating requirements.txt file. It would be a nice addition to the library.

**Comment by jonafato** [comment]
Agreed that this would be a nice addition. Closing as a duplicate of #51.
